professional-services/README.md

```diff
@@ -0,0 +1,43 @@
+# Professional Services @ DVELP
+
+Our playbook for building bespoke client-solutions at DVELP.
+
```

Comment:
@ripixelcodes let's remove these WIP parts, just start small and we can add in new parts after?